libs/inspecjs/src/nist.ts

```diff
@@ -2,9 +2,10 @@ import {ControlStatus} from './compat_wrappers';
 import {ALL_NIST_CONTROL_NUMBERS, ALL_NIST_FAMILIES} from './raw_nist';
 
 // Regexes.
-const NIST_FAMILY_RE = /^[A-Z]{2}$/;
+// Matches against only currently existing NIST Control families instead of allowing any two adjacent uppercase letters
```

Comment:
<img width="601" alt="image" src="https://github.com/mitre/heimdall2/assets/2015821/a1c8ac5f-efee-4c3c-a1ef-00442b0a5934">

seems like it is no longer falsely validating sv numbers as controls but

<img width="697" alt="image" src="https://github.com/mitre/heimdall2/assets/2015821/71e08322-ccd6-4743-9973-1dfa59a49eef">

these undefineds are getting passed through which ain't good cause they're not valid json.  idk where they're coming from tho - don't *think* it should have anything to do with the change that i made

Reply:
I don't see `undefined` in any of the `libs/hdf-converters/sample_jsons/xccdf_results_mapper/*.json` files. Can you link me to where I can see it or provide the file name that I can look at to see this problem?

Reply:
I got these by doing the yarn run test.  you can see them in the logs on github as well.

Reply:
I don't see this in any of the github logs, nor in the output of `yarn test`, nor or any of the files.

Is it possible you're looking at an old build or old code?

If you can find it, can you please link me to it, or give me the path of a file and line number to took at?

Reply:
Realized what was happening: the test is comparing the output of the file (JSON) with the output from running the toHdf function (JS).  consequently it sees the 'attribute: unknown' in there cause that's still valid in js (and is functionally a no-op) but it gets removed when turned into json.  when you get rid of any other differences in the files (ex. the differently formatted / duplicate nist tags), then it seems like jest is smart enough to realize that they are equal and doesn't surface anything into the console.